src/hooks/useClaimReward.ts

```diff
@@ -112,18 +113,19 @@ export default function useClaimReward() {
         .then((res: any) => {
```

Comment:
why do we need 2 `.then` here?
btw, can you add type for response from contract?